sys/ztimer/core.c

```diff
@@ -82,14 +82,15 @@ void ztimer_remove(ztimer_clock_t *clock, ztimer_t *timer)
     irq_restore(state);
 }
 
-void ztimer_set(ztimer_clock_t *clock, ztimer_t *timer, uint32_t val)
+uint32_t ztimer_set(ztimer_clock_t *clock, ztimer_t *timer, uint32_t val)
 {
     DEBUG("ztimer_set(): %p: set %p at %" PRIu32 " offset %" PRIu32 "\n",
           (void *)clock, (void *)timer, clock->ops->now(clock), val);
 
     unsigned state = irq_disable();
 
-    _ztimer_update_head_offset(clock);
+    uint32_t now = _ztimer_update_head_offset(clock);
+
```

Comment:
here

Reply:
Maybe you added it while I a was still presented with an old state :)